### DISCUSSION TITLE
Asking irrelevant questions with codebase on Copliot Chat

### DISCUSSION THREAD

**Discussion by javadsalman** [community/community#67295] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

When I ask an irrelevant question with the codebase, it answers. But when I keep the conversation going with short answers it tries to answer the question based on the file I currently open on vscode.

**Comment by mehulrajdev** [comment]
ok

**Comment by Akash1134** [comment]
Hi @javadsalman ,

Could you please elaborate what are your prompts and are you trying to see how far Copilot Chat can answer your inputs?

Please note that, depending on your scenario and input data, you could experience different levels of performance. While not limited specifically, GitHub Copilot Chat has [limitations](https://docs.github.com/en/copilot/github-copilot-chat/about-github-copilot-chat#limitations-of-github-copilot-chat), which may include:

- Potential inaccuracies and irrelevant suggestions
- Lack of context awareness beyond the provided code context
- Lack of deep domain knowledge or expertise in niche programming languages
- Biases present in the training data that can influence the suggestions
- The code suggestions can be rendered vulnerable due to vulnerabilities present in the training data

We recommend that you should be cautious of security and privacy concerns and avoid sharing sensitive information. It is important to exercise judgment, review suggestions, and ensure the generated code aligns with specific requirements and best practice.
